packages/types/src/event.ts

```diff
@@ -33,6 +34,7 @@ export interface Event {
   extra?: { [key: string]: any };
   user?: User;
   type?: EventType;
+  threads?: Thread[];
```

Comment:
Please add a `@deprecated Use event.threads.0.stacktrace` comment to `stacktrace`.